### PR TITLE
Update JuMP-dev 2023 details

### PR DESCRIPTION
**Pull request by odow**
Let me confirm the plan with Ranjan in my call tomorrow morning (12:00 eastern) before we publish this.

Updated form link is https://forms.gle/1pWBhu3VwBUophFV7. Please make/suggest changes/comments.

### CODE REVIEW

File: _posts/2022-11-20-jump_dev_2023.md [hunk 1]
```diff
@@ -19,32 +18,35 @@ practitioners with interests in the methodological, algorithmic, and software as
...
 
-We particularly encourage people who have not been to a JuMP-dev before to
-apply.
+* All day: talks
 
-[Apply for travel support](https://forms.gle/1pWBhu3VwBUophFV7)
+#### July 29
 
-The deadline for applications is January 31st.
+ * 0900-1400: hackathon
```
Comment (blegat) [inline _posts/2022-11-20-jump_dev_2023.md:36]:
```suggestion
 * 9h-14h: hackathon
```

**Comment by blegat** [comment]
For the question "Do you have any relevant code or academic work that is publicly available? What is your GitHub and/or Julia discourse handle?", what should the participant answer if he has code but no Github handle ?

File: _posts/2022-11-20-jump_dev_2023.md (hunk 1 above)
Comment (odow) [reply _posts/2022-11-20-jump_dev_2023.md]: Ha. Adding an `h` after makes it even more confusing to NZ speakers. `09:00 - 14:00` or `9am - 2pm`?

File: _posts/2022-11-20-jump_dev_2023.md [hunk 2]
```diff
@@ -19,32 +19,46 @@ practitioners with interests in the methodological, algorithmic, and software as
 [JuMP](https://github.com/jump-dev/JuMP.jl) and related packages. In particular,
 we invite new contributors and those who have not met the core development team.
 
-## How do I attend?
+## Schedule
 
-To attend JuMP-dev, you must purchase a ticket to JuliaCon 2023. Ticket sales
-will open in early 2023.
+A preliminary schedule is as follows. The full JuliaCon runs from July 25-29. By
+purchasing a ticket to JuMP-dev, you can also attend the full JuliaCon
```
Comment (mlubin) [inline _posts/2022-11-20-jump_dev_2023.md:25]: The wording "purchasing a ticket to JuMP-dev" is a bit confusing given that below we talk about purchasing a ticket to JuliaCon. The tickets are for JuliaCon.

Comment (odow) [reply _posts/2022-11-20-jump_dev_2023.md]: I've changed the sentence.

**Review by mlubin — APPROVED** [review]

File: _posts/2022-11-20-jump_dev_2023.md (hunk 1 above)
Comment (chriscoey) [reply _posts/2022-11-20-jump_dev_2023.md]: 9am - 2pm is the least pretentious 😄

**Comment by chriscoey** [comment]
in my browser, the form https://forms.gle/1pWBhu3VwBUophFV7 seems to lack consistent vertical spacing before the "Value of the awards" section title.

File: _posts/2022-11-20-jump_dev_2023.md [hunk 3]
```diff
@@ -19,32 +19,45 @@ practitioners with interests in the methodological, algorithmic, and software as
...
 
 ## How do I give a talk?
 
-**To give a talk, [submit a proposal to JuliaCon](https://pretalx.com/juliacon2023/cfp)**
-**For "Session type," choose "Talk" or "Lightning talk." For "Track," choose**
-**"JuMP."**
+**To give a talk, [submit a proposal](https://forms.gle/1pWBhu3VwBUophFV7)**
 
-The deadline for talk submissions is January 15, 2023.
+The deadline for talk submissions is Feburary 28, 2023.
```
Comment (chriscoey) [inline _posts/2022-11-20-jump_dev_2023.md:60]:
```suggestion
The deadline for talk submissions is February 28, 2023.
```

File: _posts/2022-11-20-jump_dev_2023.md [hunk 4]
```diff
@@ -19,32 +19,45 @@ practitioners with interests in the methodological, algorithmic, and software as
...
-[Apply for travel support](https://forms.gle/1pWBhu3VwBUophFV7)
+#### July 29
 
-The deadline for applications is January 31st.
+ * 09:00-14:00: JuMP-dev hackathon
+
+## How do I attend?
+
+To attend JuMP-dev, you must purchase a ticket to JuliaCon 2023. Ticket sales
+will open in early 2023. Accepted speakers at JuMP-dev will receive a free
```
Comment (chriscoey) [inline _posts/2022-11-20-jump_dev_2023.md:51]: we don't have an estimated date/month for tickets going on sale?

File: _posts/2022-11-20-jump_dev_2023.md (hunk 1 above)
Comment (odow) [reply _posts/2022-11-20-jump_dev_2023.md]: Is 24-hr time pretentious???

File: _posts/2022-11-20-jump_dev_2023.md (hunk 4 above)
Comment (odow) [reply _posts/2022-11-20-jump_dev_2023.md:51]: Nope. I assume in the next couple of weeks. I think they need to co-ordinate with NumFOCUS, which might take a bit of time.

**Comment by odow** [comment]
> seems to lack consistent vertical spacing

Fixed. There was a weird formatting thing in the editor. The live preview didn't match the actual preview, so I added an extra blank line.

File: _posts/2022-11-20-jump_dev_2023.md (hunk 1 above)
Comment (chriscoey) [reply _posts/2022-11-20-jump_dev_2023.md]: haha it's fine. people in america don't use the `h` Benoit suggested though

**Review by chriscoey — APPROVED** [review]

File: _posts/2022-11-20-jump_dev_2023.md (hunk 1 above)
Comment (blegat) [reply _posts/2022-11-20-jump_dev_2023.md]: 9am - 2pm is fine then :laughing:

**Review by blegat — APPROVED** [review]

File: _posts/2022-11-20-jump_dev_2023.md [hunk 5]
```diff
@@ -19,32 +19,45 @@ practitioners with interests in the methodological, algorithmic, and software as
...
 
-We particularly encourage people who have not been to a JuMP-dev before to
-apply.
+* All day: JuMP-dev talks
 
-[Apply for travel support](https://forms.gle/1pWBhu3VwBUophFV7)
+#### July 29
 
-The deadline for applications is January 31st.
+ * 09:00-14:00: JuMP-dev hackathon
```
Comment (odow) [inline _posts/2022-11-20-jump_dev_2023.md:46]:
```suggestion
 * 9am-2pm: JuMP-dev hackathon
```

**Pull request merged**